packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -377,7 +379,7 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
           }
 
           // prepend base (dev base is guaranteed to have ending slash)
-          url = base + url.replace(/^\//, '')
+          url = joinUrlSegments(base, url.replace(/^\//, ''))
```

Comment:
I believe the `.replace` call is stripping a leading slash, but that's no longer needed if you use `joinUrlSegments` because that method will handle slash or no slash appropriately in either case
```suggestion
          url = joinUrlSegments(base, url)
```